woocommerce-payments.php

```diff
@@ -98,6 +99,52 @@ function wcpay_init() {
 // If you change the priority of this action, you'll need to change it in the wcpay_check_old_jetpack_version function too.
 add_action( 'plugins_loaded', 'wcpay_init', 11 );
 
+/**
+ * Check WC Payments version and run update related actions.
+ */
+function wcpay_check_version() {
+	if ( defined( 'IFRAME_REQUEST' ) ) {
```

Comment:
Is this to prevent running the code in the Customizer or is it because of something else?

Reply:
I actually copied the version check method from WC Admin. I believe IFRAME_REQUEST gets defined for all inlineframe requests. I think we don't need this anymore since I will you the [method](https://github.com/Automattic/woocommerce-payments/blob/0be8aea7400b5947b7fde3bb739ac8448ea270f9/includes/class-wc-payments.php#L765) you suggested. I will remove it 👍